src/components/MultiHopTrade/hooks/useGetTradeQuotes/getTradeQuoteInput.ts

```diff
@@ -109,7 +109,8 @@ export const getTradeQuoteInput = async ({
     }
 
     case CHAIN_NAMESPACE.Utxo: {
-      if (!(quoteOrRate === 'quote' && wallet))
+      // This is a quote without a wallet, we monkey-patch things to the best of our ability
+      if (quoteOrRate === 'rate' && !receiveAddress)
```

Comment:
Shoudl probably keep on using the wallet discriminator here, we're interested in xpub, not receiveAddress